infra_old/environments/prod/variables.tfvars

```diff
@@ -0,0 +1,9 @@
+imms_account_id          = "084828561157"
```

Comment:
Any reason for having the prod vars here? We couldn't deploy this TF to prod as everything's hard-coded to int